README.md

```diff
@@ -4,10 +4,13 @@
 
 At the moment there wasn't a Docker image with the Z-way server available. This repo contains the steps and files to build one yourself.
 
+## Requirements
+Docker >= 1.9
+
 ## Using the image from dockerhub
 ```
 docker pull wattx/z-way-server-ubuntu:2.2.2
-docker run -it --rm --privileged -p 8083:8083 wattx/z-way-server-ubuntu:2.2.2
+docker run -it --privileged -v config-zway:/home/user/app -p 8083:8083 wattx/z-way-server-ubuntu:2.2.2
```

Comment:
if you put `--rm` it removes and container and volume? When `docker rm $(docker ps -q -f status=exited)` removes only container?